middleware/src/middleware.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/btcsuite/btcd/rpcclient"
+	system "github.com/digitalbitbox/bitbox-base/middleware/src/system"
```

Comment:
Nit: I don't think the `system` alias is necessary, now that the package is also called `system` this should be the name regardless.

Reply:
Addressed.